/aruco_node:
  ros__parameters:
    marker_size: 0.055
    aruco_dictionary_id: "DICT_5X5_250"
    camera_index: 0
    camera_destination_index: 10